.github/workflows/examples-linux-tv-casting-app.yaml

```diff
@@ -61,7 +61,7 @@ jobs:
             - name: Build Linux tv-casting-app
               run: |
                   ./scripts/run_in_build_env.sh \
-                    "scripts/examples/gn_build_example.sh examples/tv-casting-app/linux/ out/tv-casting-app"
+                    "scripts/examples/gn_build_example.sh examples/tv-casting-app/linux/ out/tv-casting-app chip_casting_simplified=true"
```

Comment:
please add chip_casting_simplified support to build_examples.py as well (change host.py to support this flag, add a variant that is conditional to targets.py)

Reply:
I have added this in the latest commit. Since I am not too familiar with this, please re-open this thread if I have missed anything. I have tested it by running `./scripts/build/build_examples.py --target darwin-arm64-tv-casting-app-chip-casting-simplified build`